xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -539,6 +552,11 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
   return settingMap;
 }
 
+string CGUIDialogSettingsBase::GetSettingsLabel(CSetting *pSetting)
```

Comment:
Is part of the DSP, is required to have the possibility to give a string direct instead the id, it can be added in the dialog and a string becomes returned for this label.
Is needed there to show addon strings.